### PR TITLE
380 sync task registry update ak requirements and measures synchronization

### PR DESCRIPTION
**Pull request by laurensWe**
# Description

- Add link to github action in the mattermost notify
- Add Error handling for conversion scripts
- Catch error handling in sync script and exit when it fails

Resolves https://github.com/orgs/MinBZK/projects/7/views/3?pane=issue&itemId=88087463&issue=MinBZK%7Camt%7C380

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

File: script/conversion/convert_mkdocs_to_yaml_requirements
Comment (ravimeijerrig) [inline script/conversion/convert_mkdocs_to_yaml_requirements:1]: Perhaps we can also change the URL to the AK in this PR: 


change line 63: 
"url": f"https://minbzk.github.io/Algoritmekader/vereisten//index.html"

to: 
"url": f"https://minbzk.github.io/Algoritmekader/voldoen-aan-wetten-en-regels/vereisten/{url_title}/index.html"

File: script/conversion/convert_mkdocs_to_yaml_measure
Comment (ravimeijerrig) [inline script/conversion/convert_mkdocs_to_yaml_measure:1]: Same here for line 57:
"https://minbzk.github.io/Algoritmekader/voldoen-aan-wetten-en-regels/maatregelen/{url_title}/index.html"

Comment (laurensWe) [reply script/conversion/convert_mkdocs_to_yaml_measure:1]: Nice, good catches!

**Review by ChristopherSpelt — APPROVED** [review]
lgtm

**Review by ravimeijerrig — APPROVED** [review]
LGTM :)

**Pull request merged**